src/main/java/edu/harvard/h2ms/service/EventServiceImpl.java

```diff
@@ -38,10 +38,10 @@ public void setEventRepository(EventRepository eventRepository) {
     public Map<String, Long> findEventCountByTimeframe(String timeframe) {
 
         List<Event> events = Lists.newArrayList(eventRepository.findAll());
-        log.info("No. of events found: " + events.size());
+        log.info("No. of events found: ", events.size());
 
         List<String> parsedTimestamps = H2msRestUtils.extractParsedTimestamps(events, timeframe);
-        log.info("Parsed " + parsedTimestamps.size() + " timestamps by " + timeframe);
+        log.info("Parsed ", parsedTimestamps.size(), " timestamps by ", timeframe);
```

Comment:
I'm pretty sure this would need to be
`log.info("Parsed {0} timestamps by {1}", parsedTimestamps.size(), timeframe);`